.npmignore

```diff
@@ -0,0 +1,3 @@
+vega/
```

Comment:
I think there are a lot of things we don't want in the package. Let's instead define a `files` entry in the `package.json`.